.github/workflows/main.yml

```diff
@@ -701,6 +710,7 @@ jobs:
           VSC_JUPYTER_NON_RAW_NATIVE_TEST: ${{ contains(matrix.test-suite, 'nonRawNotebook') }}
           VSC_JUPYTER_CI_RUN_JAVA_NB_TEST: ${{ contains(matrix.test-suite, 'conda') }}
           VSC_JUPYTER_CI_TEST_VSC_CHANNEL: 'insiders'
+          VSC_JUPYTER_WEBVIEW_TEST_MIDDLEWARE: 'true'
```

Comment:
Did you intend to add this?

Reply:
Yup, test was failing on CI as I'd only added this to launch.json. This forces on the test middleware. Gives me the variables_complete test event that I'm watching for to see when the variables are done rendering.